code/datums/keybinding/shell.dm

```diff
@@ -0,0 +1,26 @@
+/datum/keybinding/shell
+	category = CATEGORY_ROBOT
+	weight = WEIGHT_ROBOT
+
+/datum/keybinding/shell/can_use(client/user)
+	var/mob/living/silicon/robot/shell/our_shell = user.mob
+	if(iscyborg(user.mob)&&our_shell.shell)
```

Comment:
Verify that the user mob is of the correct type, add spacing between the &&'s

Reply:
alright i spread it out so it verifies if the mob is of the correct type before creating the our_shell variable